### PR TITLE
fix: QSP miscellaneous fixes

### PR DESCRIPTION
**Pull request by 0xGrapefruit**

### CODE REVIEW

File: contracts/ControllerRegistry.sol [hunk 1]
```diff
@@ -9,11 +10,24 @@ contract ControllerRegistry is IControllerRegistry, Ownable {
...
+    {
+        return controllerRegistry[_controller];
+    }
+
     /**
      * @param _controller The address to register as a controller
      */
     function registerController(address _controller) external onlyOwner {
-        require(_controller != address(0), "Invalid address");
+        require(Address.isContract(_controller), "controller was not contract");
```
Comment (jtlacci) [inline contracts/ControllerRegistry.sol:30]: this will fail on `address(0)` as well correct?

**Review by jtlacci — APPROVED** [review]
Can you document in the PR which QSPs these fix

**Comment by 0xGrapefruit** [comment]
@jtlacci These aren't specific QSPs, they're a list of fixes they recommended at the bottom of the report.

File: contracts/ControllerRegistry.sol (hunk 1 above)
Comment (0xGrapefruit) [reply contracts/ControllerRegistry.sol:30]: Yep, unless someone manages to deploy a contract to the 0 address in which case the whole world will explode.

**Pull request merged**